users/delete_handler.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Add a description of this cron job explaining what it does and why. Keep in mind that scaling Users svc will result in scaling this cron, which is not ideal, and we will probably extract this to a separate process in the future.